src/components/MediaSettings/MediaSettings.vue

```diff
@@ -732,6 +742,10 @@ export default {
 		align-items: center;
 		justify-content: center;
 		gap: var(--default-grid-baseline);
+		position: sticky;
+		bottom: 0;
+		background-color: var(--color-main-background);
+		padding: 10px 0;
```

Comment:
```suggestion
		padding: 10px 0 20px;
```
together with
```css
.media-settings {
    padding: calc(var(--default-grid-baseline)* 5);
    padding-bottom: 0;
```
should make it more fixed (now it's jumping between 10 px and 20px from the bottom)


Also NcSelect for some reason is clickable through the overlay. Fixable with `z-index: 1` though
![image](https://github.com/nextcloud/spreed/assets/93392545/548682e3-218f-49b2-8b1e-3c9bade71b4c)